autogpt_platform/backend/schema.prisma

```diff
@@ -33,6 +33,8 @@ model User {
   notifyOnDailySummary         Boolean @default(true)
   notifyOnWeeklySummary        Boolean @default(true)
   notifyOnMonthlySummary       Boolean @default(true)
+  
+  timezone                     String  @default("not-set")
```

Comment:
The datetime we are storing is timezone supported we should not do this, we just need to make changes on the application side to not always persist timezone if we really2 need. Though, why can't we just convert everything to UTC as we currently do and let all the timzone conversion happening in the frontend? The frontend knows about the local timezone and that don't need to be stored.